content/en/news/releases/1.7.x/announcing-1.7.2/index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Announcing Istio 1.7.2
+linktitle: 1.7.2
+subtitle: Patch Release
+description: Istio 1.7.2 patch release.
+publishdate: 2020-09-18
+release: 1.7.2
+aliases:
+    - /news/announcing-1.7.2
+---
+
+This release contains bug fixes to improve robustness. This release note describes what’s different between Istio 1.7.1 and Istio 1.7.2
+
+{{< relnote >}}
+
+## Changes
+
+- **Fixed** a bug that locality load balancer settings were applied to inbound clusters unnecessarily. ([Issue #27293](https://github.com/istio/istio/issues/27293))
```

Comment:
Maybe **Fixed** locality load balancer settings being applied to inbound clusters unnecessarily.

Reply:
Done. Thanks!